common/src/main/java/com/microsoft/identity/common/internal/telemetry/TelemetryEventStrings.java

```diff
@@ -0,0 +1,189 @@
+    public static final String TELEMETRY_KEY_UI_CANCELLED = "ui_cancelled";
+    public static final String TELEMETRY_KEY_SERVER_ERROR_CODE = "server_error_code";
+    public static final String TELEMETRY_KEY_SERVER_SUBERROR_CODE = "server_sub_error_code";
+    public static final String TELEMETRY_KEY_RT_AGE = "rt_age";
+    public static final String TELEMETRY_KEY_SPE_INFO = "spe_info";
+    public static final String TELEMETRY_KEY_SPE_RING = "spe_ring";
+    public static final String TELEMETRY_KEY_IS_SUCCESSFUL = "is_successful";
+    public static final String TELEMETRY_KEY_WIPE_APP = "wipe_app";
+    public static final String TELEMETRY_KEY_WIPE_TIME = "wipe_time";
+
+
+    // App
+    public static final String TELEMETRY_KEY_APP_BUILD = "app_build";
+    // Device
+    public static final String TELEMETRY_KEY_DEVICE_MANUFACTURER = "device_manufacturer";
+    public static final String TELEMETRY_KEY_DEVICE_MODEL = "device_model";
+    public static final String TELEMETRY_KEY_DEVICE_NAME = "device_name";
+    public static final String TELEMETRY_KEY_DEVICE_TIMEZONE = "time_zone";
+    // OS
+    public static final String TELEMETRY_KEY_OS_NAME = "os_name";
+    public static final String TELEMETRY_VALUE_OS_NAME = "android";
+    public static final String TELEMETRY_KEY_OS_VERSION = "os_version";
+    public static final String TELEMETRY_KEY_TIMEZONE = "timezone";
+
+    // Telemetry property value
+    public static final String TELEMETRY_VALUE_TRUE = "true";
+    public static final String TELEMETRY_VALUE_FALSE = "false";
+    public static final String TELEMETRY_VALUE_TRIED = "tried";
+    public static final String TELEMETRY_VALUE_USER_CANCELLED = "user_cancelled";
+    public static final String TELEMETRY_VALUE_NOT_FOUND = "not_found";
+    public static final String TELEMETRY_VALUE_ACCESS_TOKEN = "access_token";
+    public static final String TELEMETRY_VALUE_REFRESH_TOKEN = "refresh_token";
+    public static final String TELEMETRY_VALUE_MULTI_RESOURCE_REFRESH_TOKEN = "multi_resource_refresh_token";
+    public static final String TELEMETRY_VALUE_FAMILY_REFRESH_TOKEN = "family_refresh_token";
+    public static final String TELEMETRY_VALUE_ADFS_TOKEN = "ADFS_access_token_refresh_token";
+    public static final String TELEMETRY_VALUE_BY_CODE = "by_code";
+    public static final String TELEMETRY_VALUE_BY_REFRESH_TOKEN = "by_refresh_token";
+    public static final String TELEMETRY_VALUE_SUCCEEDED = "succeeded";
+    public static final String TELEMETRY_VALUE_FAILED = "failed";
+    public static final String TELEMETRY_VALUE_CANCELLED = "cancelled";
+    public static final String TELEMETRY_VALUE_UNKNOWN = "unknown";
+    public static final String TELEMETRY_VALUE_AUTHORITY_AAD = "aad";
+    public static final String TELEMETRY_VALUE_AUTHORITY_ADFS = "adfs";
+    public static final String TELEMETRY_VALUE_AUTHORITY_B2C = "b2c";
+
+    //Telemetry API ID
+    public static final String API_BROKER_ACQUIRE_TOKEN_INTERACTIVE = "201";
```

Comment:
What was the reason of sending number instead of actual strings? (as this would require a mapping on the other side)

Reply:
Shorten the blob we sent. This is the same with ADAL telemetry as well as the iOS telemetry.